.github/workflows/test.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
     - run: npm install
-    - run: npm start & sleep 5 && npm test
+    - run: npm test
```

Comment:
https://user-images.githubusercontent.com/38065883/175115387-64b84a18-ad7e-466d-a575-91181ed98292.mov

- initially page isn't scrollable, cause of with-menu class
- remove standard border from submit form button